server/document-watch.worker.js

```diff
@@ -26,7 +26,15 @@ function postDocumentInfo(filePath, changeType) {
     // matches the filePath by using `Document.urlToFolderPath`.
     // This would prevent the document from being added in the first place
     // if the filePath doesn't map correctly to the URL, but in reverse.
-    if (!filePath.includes(Document.urlToFolderPath(document.url))) {
+    // Note! On Windows, `glob.sync()` will return URLs that always use
+    // the `/` character (POSIX STYLE, https://www.npmjs.com/package/glob#windows)
+    // But the `Document.urlToFolderPath()` function will respect Windows
+    // notation and use `\` characters.
+    if (
+      !filePath.includes(
+        Document.urlToFolderPath(document.url).replace(/\\/g, "/")
```

Comment:
I'd suggest we transform `filePath` before we invoke `postDocumentInfo`. I'm curious why the `Document.read` works in this case it it does. Isn't `filePath` something like `en-us/web/css/index.html` on a windows system but `Document.read` expects `en-us\web\css\index.html` ?

I can test it on windows with some effort, should I?

Reply:
So, you suggest the other way around. I think I get it. That way, the `filePath` gets Windows'ified onwards. I'll look into it.